src/main/kotlin/com/stefan_grafberger/streamdq/anomalydetection/detectors/AnomalyCheck.kt

```diff
@@ -0,0 +1,25 @@
+package com.stefan_grafberger.streamdq.anomalydetection.detectors
+
+import com.stefan_grafberger.streamdq.anomalydetection.strategies.AnomalyDetectionStrategy
+import org.apache.flink.streaming.api.windowing.assigners.WindowAssigner
+import org.apache.flink.streaming.api.windowing.windows.TimeWindow
+
+/**
+ * The interface for building AnomalyDetector
+ * It is used for adding all the needed configurations
+ * when user define anomaly check before using it in the
+ * [com.stefan_grafberger.streamdq.VerificationSuite]
+ *
+ * @Override by [com.stefan_grafberger.streamdq.anomalydetection.detectors.aggregatedetector]
+ *
+ * @since 1.0
```

Comment:
Hi Tong, would it be okay to remove these versioning annotations everywhere? I think they only make sense if they are used throughout a whole project and if they are actually maintained, and I don't think this is worth the effort currently :)

Reply:
Yes sure Stefan, will do this